README.md

```diff
@@ -1,42 +1,37 @@
+that let you access and transform simulation data coming from different Ansys and third-party
+result files.
 
-The Python `ansys-dpf-post` package provides a high-level, physics-oriented
-API for postprocessing. Loading a simulation (defined by its results files)
-allows you to extract simulation metadata and results and then apply
-postprocessing operations on them.
-
-The latest version of DPF supports Ansys solver results files for:
-
-  - Mechanical APDL (`.rst`, `.mode`, `.rfrq`, `.rdsp`)
-  - LS-DYNA (`.d3plot`, `.binout`)
-  - Fluent (`.cas/dat.h5`, `.flprj`)
-  - CFX (`.cas/dat.cff`, `.flprj`, `.res`)
-
-For more information on file support, see the [main page](https://dpf.docs.pyansys.com/version/stable/index.html)
-in the PyDPF-Core documentation.
-
-PyDPF-Post leverages the PyDPF-Core project's ``ansys-dpf-core`` package, which is
-available at [PyDPF-Core GitHub](https://github.com/ansys/pydpf-core).
-Use the ``ansys-dpf-core`` package for building more advanced and customized workflows
-using Ansys DPF.
+PyDPF-Post leverages [PyDPF-Core](https://github.com/ansys/pydpf-core), a package for building more advanced and customized workflows using Ansys DPF. After loading a simulation's result fiile in PyDPF-Post, you can extract simulation metadata and results and then apply postprocessing operations on them.
 
 ## Documentation and issues
 
 Documentation for the latest stable release of PyPDF-Post is hosted at
 [PyDPF-Post documentation](https://post.docs.pyansys.com/version/stable/).
 
-In the upper right corner of the documentation's title bar, there is an option for switching from
-viewing the documentation for the latest stable release to viewing the documentation for the
-development version or previously released versions.
+The PyDPF-Post documentation has five sections:
+
+- [Getting started](https://post.docs.pyansys.com/version/stable/getting_started/index.html): Learn how to
+  install PyDPF-Post in user mode and quickly begin using it.
+- [User guide](https://post.docs.pyansys.com/version/stable/user_guide/index.html): Understand key concepts
+  for using PyDPF-Post.
+- [API reference](https://post.docs.pyansys.com/version/stable/api/index.html): Understand how to use
+  Python to interact programmatically with PyDPF-Post.
+- [Examples](https://post.docs.pyansys.com/version/stable/examples/index.html): Explore examples
+  that show how to use PyDPF-Post to perform nay different types of operations.
+- [Contribute](https://post.docs.pyansys.com/version/stable/contributing.html): Learn how to
+  contribute to the PyDPF-Post codebase or documentation.
+
+In the upper right corner of the documentation's title bar, there is an option
+for switching from viewing the documentation for the latest stable release
+to viewing the documentation for the development version or previously
+released versions.
 
 You can also [view](https://cheatsheets.docs.pyansys.com/pydpf-post_cheat_sheet.png) or
```

Comment:
@PProfizi Do we want to remove cheat sheet information here and display in the left column of the doc landing page--such as has been done on other doc landing pages (such as PyMechanical's as shown below and PyFluent's)? @Revathyvenugopal162 Based on [this Cheat Sheet](https://sphinxdocs.ansys.com/version/stable/user-guide/options.html#cheat-sheets) topic in the Ansys Sphinx Theme documentation, is the ability to do this now available only if you are using Quarto?

Reply:
> @PProfizi Do we want to remove cheat sheet information here and display in the left column of the doc landing page--such as has been done on other doc landing pages (such as PyMechanical's as shown below and PyFluent's)? @Revathyvenugopal162 Based on [this Cheat Sheet](https://sphinxdocs.ansys.com/version/stable/user-guide/options.html#cheat-sheets) topic in the Ansys Sphinx Theme documentation, is the ability to do this now available only if you are using Quarto?

yes,  you need to have quarto installed.